src/modules/iotjs_module_mqtt.c

```diff
@@ -153,6 +153,20 @@ JS_FUNCTION(MqttGetConnect) {
   return retbuf;
 }
 
+
+void alloc_payload_buf(unsigned char **buf, int expected_size, int *alloc_size) {
+  static int buf_size = 1 * 1024 * 1024;
+  static unsigned char *buf_ = NULL;
```

Comment:
Why do we here use `static` variable? Stateless function here would be more clearable :)